go/nbs/store.go

```diff
@@ -381,19 +380,12 @@ func (nbs *NomsBlockStore) Commit(current, last hash.Hash) bool {
 		return true
 	}
 
-	b := &backoff.Backoff{
```

Comment:
I'm not sure what happens in that case. My impression is that we get transparently throttled. If we also get errors, then it may be reasonable to back off.

https://github.com/attic-labs/noms/issues/3535